lang/php/Portfile

```diff
@@ -2111,28 +2099,31 @@ if {[is_extension_subport]} {
                 ln -s ${worksrcpath}/ext/pdo ${worksrcpath}/ext/${extension}/ext
             }
         }
-        
+
         # Speed up extraction by extracting only the modules we're going to be building.
         extract.post_args-append ${worksrcdir}/ext/${extension}
-        
+
         # Run phpize, configure, build and destroot in each extension's directory.
         php.build_dirs-append ${worksrcpath}/ext/${extension}
     }
-    
+
     if {${pdo}} {
         # The PDO extensions need the PDO headers.
         extract.post_args-append ${worksrcdir}/ext/pdo
     }
-    
+
     pre-configure {
         set php_version [exec ${php.config} --version 2>/dev/null]
         if {${version} ne ${php_version}} {
-            ui_error "${subport} @${version} requires ${php} @${version} but you have ${php} @${php_version}."
+            ui_error "
+${subport} @${version} requires ${php} @${version} but you have\
+${php} @${php_version}.
+            "
```

Comment:
This commit will be extremely difficult to review due to its size, so I'll just start with this "reformatting" of ui_error, which is not merely a reformatting: it changes the output that the user will see (introducing a newline before and after the error message). I don't really want that; that's not something I'm aware of us doing in any other port. Let's leave ui_errors formatted the way they were.

Reply:
I'll fix that…

Reply:
Done √